mobile-workspace/latest/using/index.md

```diff
@@ -154,32 +153,82 @@ Filters can be set for Status, Due date, and Task Name. Filter chips are shown a
+3. Press **Apply** to set the filter and **Reset** to reset the filter to default. The **X** in the top right closes the window and returns you to the Task list.
 
 4. The filter chips will show the active filters and the refresh button will reset the filters.
 
 ### Task view
 
-Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit Task view and return to the Task list.
+Within the Task view are details about the task: Task name, Due date, Priority, Assignee, Status, and Identifier. There is a back arrow to exit the Task view and return to the Task list.
 
-Individual tasks have a “Comments” section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below existing comments.
+Individual tasks have a Comments section below the task details. The most recent comment is shown with the user's icon, user name, date the comment was added, and the user’s comment. If there is more than one comment, you will see a link to **View all**. The link **Add a comment…** is below the existing comments. If you have access to  
 
 Attached files is a section below Comments that shows the number of attached files to the task. The link **View all** displays an Attached files page when there are more than four attachments. Clicking an attachment will download the file to preview with a popup and **Cancel** option.
 
 ### Complete Tasks
 
-You will be able to complete tasks if the task is assigned to you.
+You can complete tasks if the task is assigned to you.
 
-1. Open a task from the Task list. At the bottom of the task is a **Complete** floating action button.
+1. Open a task from the Task list.
 
-2. Press the **Complete** button.
+    At the bottom of the task is a **Mark as Complete** floating action button.
 
-    A popup is displayed: `Are you sure you want to complete the task? You will no longer be able to make any changes.`
+2. Press the **Mark as Complete** button.
+
+    A popup is displayed: **Are you sure you want to complete the task? You will no longer be able to make any changes.**
 
     A completed task can be viewed, but enters a read-only mode. No comments or attachments can be added to a completed task.
 
 3. Press **Confirm** to complete the task. Press **Cancel** to return to the task.
+
+![Complete task]({% link mobile-workspace/images/mark-complete.png %})
+
+### Reassign task
+
+You can reassign a task to another person.
+
+1. Open a task from the Task list.
+
+2. Press **Edit** on the top right.
+
+3. Press the **Edit** icon next to the **Assignee** name.
+
+4. Search for a new assignee using their name or email address.
```

Comment:
you need to enter their full email address

Reply:
I dont think any changes need to be made here, I have been through the process again now. You can search by name and email address. I dont need to mention full email address as that doesnt make any sense to me :-)